util/check-format-commit.sh

```diff
@@ -12,24 +12,9 @@
 # fall into the commits change ranges.
 #
 
-
-# List of Regexes to use when running check-format.pl.
-# Style checks don't apply to any of these
-EXCLUDED_FILE_REGEX=("\.pod" \
-                     "\.pl"  \
-                     "\.pm"  \
-                     "\.t"   \
-                     "\.yml" \
-                     "\.sh" \
-                     "\.cnf" \
-                     "\.conf" \
-                     "\.info" \
-                     "\.md" \
-                     "\.S" \
-                     "\.pem" \
-                     "\.txt" \
-                     "\.dat" \
-                     "Configure") 
+# Allowlist of files to scan
+# Currently this is any .c or .h file (with an optional .in suffix
+FILE_ALLOWLIST=("\.[ch]\(.in\)\?")
```

Comment:
This will catch anything with `.c` or `.h` in the filename, even without the `.in` part - i.e. would match `.c.bak`. Should there be a `$` here? (and if not, we don't need the `.in` part)

Reply:
the input file is of the format:
```
<commit> <file path> <range start>,<range length>
```
so we can't use a $ here directly.

We could do grep "\.[ch]\(.in\)\? "
i.e. with a space following the suffix to ensure we don't catch anything like .cms .  Would that work?

Reply:
... and we can't even use word boundary, due to `.`. So yes to the space - but I would put the space in the `grep`, with a comment above it about why we are doing it (if that makes sense)